src/code/code_800F9280.c

```diff
@@ -2,9 +2,88 @@
 #include "global.h"
 #include "ultra64/abi.h"
 
-extern u8 sSeqCmdRdPos;
-extern u8 sSeqCmdWrPos;
-extern u8 D_80133410[];
+// bss
```

Comment:
the comment here isn't really needed anymore since this is now imported.

Reply:
I manually put it in there just for labelling/docing the data, but yeah I can take it out